robot_sf/gym_env/empty_robot_env.py

```
@@ -34,11 +35,11 @@ class EmptyRobotEnv(Env):
     """
 
     def __init__(
-            self,
-            env_config: EnvSettings = EnvSettings(),
-            reward_func: Callable[[dict], float] = simple_reward,
-            debug: bool = False
-            ):
+        self,
+        env_config: EnvSettings = EnvSettings(),
```

Ruff

Do not perform function call `EnvSettings` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)